caikit/runtime/http_server/http_server.py

```diff
@@ -498,10 +506,25 @@ async def _generator() -> pydantic_response:
                             **request_params,
                         )
                     ):
-                        yield result
+                        yield {
+                            "event": StreamEventTypes.MESSAGE.value,
+                            "data": result.to_json(),
```

Comment:
Hmm, I guess a single line str is easier to parse vs multi-line `data: dict` statements? Because without this `to_json`, we get
```
data: {
data:   "greeting": "Hello world stream"
data: }
```

That's what the internal team wanted us to do, which made sense to me. We can discuss this further

Reply:
Also note that yielding just the `result` part back _still_ sends in something like this:
```
data: {
data:   "greeting": "Hello world0 stream"
data: }
```
Note that `data` is still part of the returned message

Reply:
Ok, I found the context I was missing. It looks like `data` and `event` are "first class things" in SSE and not just random strings we happen to be using, so my concern about double-decoding is invalid

CITE: https://github.com/sysid/sse-starlette/blob/159bbece1a58434d12d67934dc99fab556737247/sse_starlette/sse.py#L50